drbd_metrics_test.go

```diff
@@ -166,22 +166,46 @@ func TestDrbdParsing(t *testing.T) {
 		t.Errorf("lower-pending should be 2")
 	}
 
+	if true != drbdDevs[0].Devices[0].Quorum {
+		t.Errorf("quorum should be true")
+	}
+
+	if true != drbdDevs[1].Devices[0].Quorum {
```

Comment:
I would suggest you change the fake drbd data to have devices with and without quorum, so we test all possibilities here

Reply:
Good call, will do!

Reply:
pushed new commit setting the second resource and expected results to `false`. Thanks!

Reply:
good point thx